src/js/generators.js

```diff
@@ -256,7 +256,7 @@ function generate_enumerate_chord() {
     if (/^\d+$/.test(value))
       value += ",";
     value = line_to_decimal(value);
-    if (value === 0 || !/(^\d+([\,\.]\d*)?|([\\\/]\d+)?$)*/.test(value)) {
+    if (value === 0 || !/(^\d+([,.]\d*)?|([\\/]\d+)?$)*/.test(value)) {
```

Comment:
Nice find!

Reply:
These regular expressions should be moved to constants in the upcoming refactors, so people who read the code will get an idea on what is happening here faster.